.gitlab/common/shared.yml

```diff
@@ -34,9 +34,3 @@
   - *get_artifactory_bypass_win
   - *get_artifactory_token_win
   - if (($Env:USE_CACHING_PROXY_PYTHON -eq "true") -and ($Env:ARTIFACTORY_BYPASS -eq "false")) { $PIP_INDEX_URL="https://${Env:ARTIFACTORY_USERNAME}:${ARTIFACTORY_TOKEN}@${Env:ARTIFACTORY_URL}/${Env:ARTIFACTORY_PYPI_PATH}" }
-
-.setup_deb_signing_key: &setup_deb_signing_key
-  - set +x
```

Comment:
hopefully there is no more `set -x` in the repo but we must take care of this kind of stuff